networking/src/Models/ITeamAnswerContent.ts

```diff
@@ -0,0 +1,29 @@
+import { GameSessionState } from "../AWSMobileApi";
+enum AnswerType {
+  MULTICHOICE,
+  TEXT,
+  NUMBER,
+  FORMULA,
+  NULL
+}
+
```

Comment:
this gets adjusted somewhat in the other PRs in this batch. INormAnswerSubObj gets removed, for instance